### PR TITLE
Document CVE fixes

### PR DESCRIPTION
**Pull request by radarhere**

### CODE REVIEW

**Comment by hugovk** [comment]
The bullet points need some tweaking:

https://pillow--5176.org.readthedocs.build/en/5176/releasenotes/8.1.0.html

image

**Comment by radarhere** [comment]
Ok, I've updated the commit

File: docs/releasenotes/8.1.0.rst [hunk 1]
```diff
@@ -44,8 +36,29 @@ already exists for the ICNS format.
 Security
 ========
 
-An out-of-bounds read when saving TIFFs with custom metadata through libtiff has been
-fixed, as well as when saving a GIF of 1px width.
+This release includes security fixes.
+
+* An out-of-bounds read when saving TIFFs with custom metadata through libtiff
```
Comment (hugovk) [inline docs/releasenotes/8.1.0.rst:41]:
```suggestion
* An out-of-bounds read when saving TIFFs with custom metadata through LibTIFF
```

File: docs/releasenotes/8.1.0.rst [hunk 2]
```diff
@@ -44,8 +36,29 @@ already exists for the ICNS format.
...
+* An out-of-bounds read when saving a GIF of 1px width
+* :cve:`CVE-2020-35653` Buffer Read Overrun in PCX Decoding.
```
Comment (hugovk) [inline docs/releasenotes/8.1.0.rst:43]:
```suggestion
* :cve:`CVE-2020-35653` Buffer read overrun in PCX decoding
```

File: docs/releasenotes/8.1.0.rst [hunk 3]
```diff
@@ -44,8 +36,29 @@ already exists for the ICNS format.
...
+
+The PCX Image decoder used the reported image stride to calculate the row buffer,
```
Comment (hugovk) [inline docs/releasenotes/8.1.0.rst:45]:
```suggestion
The PCX image decoder used the reported image stride to calculate the row buffer,
```

File: docs/releasenotes/8.1.0.rst [hunk 4]
```diff
@@ -44,8 +36,29 @@ already exists for the ICNS format.
...
+rather than calculating it from the image size. This issue dates back to the PIL fork.
+Thanks to Google's OSS-Fuzz project for finding this.
```
Comment (hugovk) [inline docs/releasenotes/8.1.0.rst:47]: We could link "Google's OSS-Fuzz project" to https://github.com/google/oss-fuzz, here and below.

File: docs/releasenotes/8.1.0.rst [hunk 5]
```diff
@@ -44,8 +36,29 @@ already exists for the ICNS format.
...
+
+* :cve:`CVE-2020-35654` Fix TIFF OOB Write error
+
+OOB Write in TiffDecode.c when reading corrupt YCbCr files in some LibTiff versions
+(4.1.0/Ubuntu 20.04, but not 4.0.9/Ubuntu 18.04). In some cases libtiff's
+interpretation of the file is different when reading in RGBA mode, leading to an Out of
+bounds write in TiffDecode.c. This potentially affects Pillow versions from 6.0.0 to
+8.0.1, depending on the version of LibTiff. This was reported through Tidelift.
```
Comment (hugovk) [inline docs/releasenotes/8.1.0.rst:55]:
```suggestion
OOB Write in TiffDecode.c when reading corrupt YCbCr files in some LibTIFF versions
(4.1.0/Ubuntu 20.04, but not 4.0.9/Ubuntu 18.04). In some cases LibTIFF's
interpretation of the file is different when reading in RGBA mode, leading to an Out of
bounds write in TiffDecode.c. This potentially affects Pillow versions from 6.0.0 to
8.0.1, depending on the version of LibTIFF. This was reported through Tidelift.
```

File: docs/releasenotes/8.1.0.rst [hunk 6]
```diff
@@ -44,8 +36,29 @@ already exists for the ICNS format.
...
+
+* :cve:`CVE-2020-35655` Fix for SGI Decode buffer overrun
+
+4 Byte Read Overflow in SGIRleDecode.c, where the code was not correctly checking the
```
Comment (hugovk) [inline docs/releasenotes/8.1.0.rst:59]:
```suggestion
4 byte read overflow in SGIRleDecode.c, where the code was not correctly checking the
```

File: docs/releasenotes/8.1.0.rst [hunk 7]
```diff
@@ -44,8 +36,29 @@ already exists for the ICNS format.
...
+offsets and length tables. Independently reported through Tidelift and Google's OSS-Fuzz.
```
Comment (hugovk) [inline docs/releasenotes/8.1.0.rst:60]: Let's link to Tidelift using our referral link

**Review by hugovk — APPROVED** [review]
Thanks! Some minor suggestions.

File: docs/releasenotes/8.1.0.rst (hunk 7 above)
Comment (radarhere) [reply docs/releasenotes/8.1.0.rst]: Ok, done

File: docs/releasenotes/8.1.0.rst (hunk 4 above)
Comment (radarhere) [reply docs/releasenotes/8.1.0.rst]: Ok, done

**Review by hugovk — APPROVED** [review]

**Pull request merged**